tools/custom-action/entrypoint.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash -l
```

Comment:
µNit: For any less common options, I prefer the more readable, longer forms, in scripts where typability doesn't matter: `--login`

Reply:
Certainly more welcoming to spell the longer form!

Reply:
@kainino0x thank you!